fastai/text/learner.py

```diff
@@ -84,18 +99,29 @@ def load_model_text(file, model, opt, with_opt=None, device=None, strict=True):
 @delegates(Learner.__init__)
 class TextLearner(Learner):
     "Basic class for a `Learner` in NLP."
-    def __init__(self, dls, model, alpha=2., beta=1., moms=(0.8,0.7,0.8), **kwargs):
+    def __init__(self,
+        dls:DataLoaders, # Text `DataLoaders`
+        model, # Pretrained Model Checkpoint or Custom Model used to train
```

Comment:
```suggestion
        model, # A standard PyTorch model
```

Stole this wording from https://docs.fast.ai/learner.html#Learner 

I think this is the same model that Learner accepts right?

Reply:
Yes. that's correct. I can use the same wording.